openjdk.test.load/src/test.load/net/adoptopenjdk/stf/TimedLoadTest.java

```diff
@@ -0,0 +1,92 @@
+/*******************************************************************************
+* Licensed under the Apache License, Version 2.0 (the "License");
+* you may not use this file except in compliance with the License.
+* You may obtain a copy of the License at
+*
+*      https://www.apache.org/licenses/LICENSE-2.0
+*
+* Unless required by applicable law or agreed to in writing, software
+* distributed under the License is distributed on an "AS IS" BASIS,
+* WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+* See the License for the specific language governing permissions and
+* limitations under the License.
+*******************************************************************************/
+
+package net.adoptopenjdk.stf;
+
+import net.adoptopenjdk.stf.extensions.core.StfCoreExtension;
+import net.adoptopenjdk.stf.extensions.core.StfCoreExtension.Echo;
+import net.adoptopenjdk.stf.environment.FileRef;
+import net.adoptopenjdk.stf.plugin.interfaces.StfPluginInterface;
+import net.adoptopenjdk.stf.processes.ExpectedOutcome;
+import net.adoptopenjdk.stf.processes.definitions.JavaProcessDefinition;
+import net.adoptopenjdk.stf.processes.definitions.LoadTestProcessDefinition;
+import net.adoptopenjdk.stf.runner.modes.HelpTextGenerator;
+import net.adoptopenjdk.loadTest.InventoryData;
+import net.adoptopenjdk.stf.environment.StfTestArguments; 
+
+public class TimedLoadTest implements StfPluginInterface {
+	String timelimitInMins;
+	int testCountMultiplier;
+	public void help(HelpTextGenerator help) throws StfException {
+		help.outputSection("MixedLoadTest runs workloads of math,mauve,nio,lang,concurrent unit tests");
```

Comment:
Is this `TimedLoadTest ` or `MixedLoadTest `?

Reply:
Good catch! That was a copy paste error. I've updated the help message.